vpc-getting-started/cleanup.yaml

```diff
@@ -56,12 +56,13 @@
           "tag:Name": "{{ route_table_name }}"
       register: route_tables
 
-    - name: Destroy the route to the internet {{ route_tables.route_tables[0].id }}
+    - name: Destroy the route to the internet
       ec2_vpc_route_table:
         vpc_id: "{{ vpcs.objects[0].id }}"
         route_table_id: "{{ route_tables.route_tables[0].id }}"
         lookup: id
         state: absent
+      when: route_tables.route_tables
```

Comment:
Would it be possible to add a short explanation of why is this condition needed? It is not a norm to have such guards in the playbooks since modules should be idempotent.

Reply:
The module itself is indepotent, but the task here assumes that `route_tables.route_tables` is a list with one entry. If that is not the case, ansible complains. Hence the condition that the list actually has meaningful contents.